### PR TITLE
Update CognitiveServices/Text-Analytics Swagger with new /entities endpoint

### PR DESCRIPTION
**Pull request by assafi**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/examples/SuccessfulEntitiesRequest.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{
+    "parameters": {
+        "Ocp-Apim-Subscription-Key": "{API key}",
+        "body": {
```
Comment (hovsepm) [inline specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/examples/SuccessfulEntitiesRequest.json]: >body [](start = 9, length = 4)

According to your swagger this parameter should be named as `input` and not body, right?

Comment (assafi) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/examples/SuccessfulEntitiesRequest.json]: No. `input` and `body` are not part of the request body. I think this is just a reference ID for the example body which has a `documents` root object. I have followed the same pattern as existing endpoints (the input for all is the same).

Comment (assafi) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/examples/SuccessfulEntitiesRequest.json]: Here's an example request body:

<code>
{
    "documents": [
                {
                    "language": "en",
                    "id": "1",
                    "text": "Microsoft released Windows 10"
                },
                {
                    "language": "en",
                    "id": "2",
                    "text": "In 1975, Bill Gates III and Paul Allen founded the company."
                }
            ]
}
</code>

Comment (hovsepm) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/examples/SuccessfulEntitiesRequest.json]: and I think all your samples are flagged as wrong
```
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Key Phrases".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Key Phrases", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Detect Language".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Detect Language".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Detect Language", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Sentiment".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Sentiment".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Sentiment", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Entities request" in operation "Entities".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Entities request" in operation "Entities".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Entities", parameter input is required in the swagger spec but is not present in the provided example parameter values.' } ] }
```

Comment (assafi) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.0/examples/SuccessfulEntitiesRequest.json]: Thanks. Should be fixed now.

**Comment by hovsepm** [comment]
@assafi please fix this issues as well:
```
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Key Phrase request" in operation "Key Phrases".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Key Phrases", parameter AzureRegion is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Detect Language request" in operation "Detect Language".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Detect Language request" in operation "Detect Language".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Detect Language", parameter AzureRegion is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Sentiment request" in operation "Sentiment".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Sentiment request" in operation "Sentiment".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Sentiment", parameter AzureRegion is required in the swagger spec but is not present in the provided example parameter values.' } ] }
error: Found errors in validating the request for x-ms-example "Successful Entities request" in operation "Entities".:
 
{ code: 'REQUEST_VALIDATION_ERROR',
  id: 'OAV109',
  message: 'Found errors in validating the request for x-ms-example "Successful Entities request" in operation "Entities".',
  innerErrors: 
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       id: 'OAV105',
       message: 'In operation "Entities", parameter AzureRegion is required in the swagger spec but is not present in the provided example parameter values.' } ] }
```

**Review by hovsepm — APPROVED** [review]

**Pull request merged**